dash-renderer/package.json

```diff
@@ -6,10 +6,10 @@
   "scripts": {
     "prepublishOnly": "rm -rf lib && babel src --extensions=\".ts,.tsx,.js,.jsx\" --out-dir lib --copy-files",
     "private::format.js-eslint": "eslint --quiet --fix .",
-    "private::format.js-prettier": "prettier --config .prettierrc --write \"src/**/*.js\"",
+    "private::format.js-prettier": "prettier --config .prettierrc --write \"src/**/*.(js|ts|tsx)\"",
     "private::format.ts": "tslint --fix --project tsconfig.json --config tslint.json",
     "private::lint.js-eslint": "eslint .",
-    "private::lint.js-prettier": "prettier --config .prettierrc \"src/**/*.js\" --list-different",
+    "private::lint.js-prettier": "prettier --config .prettierrc \"src/**/*.(js|ts|tsx)\" --list-different",
```

Comment:
Apply Prettier to TypeScript files too.
@alexcjohnson Since you've mentioned it before. Might as well.. makes little sense to do one extension and not the other.